app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -54,6 +54,7 @@ def custom_index_relation(collection)
       param :deb_architectures, String, :desc => N_("comma separated list of architectures to be synched from deb-archive")
       param :ignore_global_proxy, :bool, :desc => N_("if true, will ignore the globally configured proxy when syncing")
       param :ignorable_content, Array, :desc => N_("List of content units to ignore while syncing a yum repository. Must be subset of %s") % RootRepository::IGNORABLE_CONTENT_UNIT_TYPES.join(",")
+      param :ansible_collection_whitelist, String, :desc => N_("Name of collection to sync from URL")
```

Comment:
Calling this `ansible_collection_whitelist` implies it should be an array to me.

Reply:
Also, the description should say `Ansible Collection` IMHO

Reply:
Aplogies, one more thing - Are you going to have a `whitelist` function for other content types with Pulp 3? If so, it would make sence to use a more generic name.

Reply:
we do also have a whitelist for docker too, thats not a bad idea!  We'd have to depreciate the old docker one, and move it over to the common one.  Although then if pulp ever supported whitelisting of multiple types within a repository, that wouldn't work too well.... hrmmmm  Let me think about it

Reply:
so i chatted with samir about this, and i initially liked the idea, but if we ever have some plugin that have multiple whitelists for content types, or some other kind of format, we'd need other parameters for that.  

I think we're going to have this explosion in repository attributes regardless that we need to work on dealing with that.  As we continue to add content types it may become apparent that this was a bad decision, and we can revisit it :)